include/Data/ByteString.spec

```diff
@@ -0,0 +1,380 @@
+module spec Data.ByteString where
+
+measure bslen :: Data.ByteString.ByteString -> { n : Int | 0 <= n }
```

Comment:
I think this line needs to be moved to `Data/ByteString/Unsafe.spec`, which should be the lowest file in the import hierarchy (`Data/ByteString/Internal.spec` would make sense too).

Right now the testsuite is failing with

```
 /home/ubuntu/liquidhaskell/include/Data/ByteString/Unsafe.spec:20:8-17: Error: Bad Type Specification
 assumed type Data.ByteString.Unsafe.unsafeTake :: n:{n : Int | 0 <= n} -> {i : ByteString | n <= bslen i} -> {o : ByteString | bslen o == n}
     Sort Error in Refinement: {i##11 : Data.ByteString.Internal.ByteString | n <= bslen i##11}
     Unbound Symbol bslen
 Perhaps you meant: plen, len, fplen
```

which makes sense as `Data.ByteString` imports `Data.ByteString.Unsafe` and thus `bslen` doesn't exist yet.